cmd/juju/commands/bootstrap_clouds.go

```diff
@@ -133,3 +136,99 @@ func printCloudRegions(ctx *cmd.Context, cloudName string) error {
 	}
 	return nil
 }
+
+func (c *bootstrapCommand) printCloudConfig(ctx *cmd.Context, showCloudConfigInput string) error {
+	// Gather the general cloud config attributes.
+	var env environschema.Fields
+	schema, _ := config.Schema(env)
+	defaults := config.ConfigDefaults()
+	values, err := configMapToPrint(schema, defaults)
+	if err != nil {
+		return err
+	}
+
+	// Gather the cloud config attributes specific to its provider.
+	var cloudType string
+	if cloud, err := common.CloudByName(showCloudConfigInput); err == nil {
+		cloudType = cloud.Type
+	} else {
+		cloudType = showCloudConfigInput
+	}
+	// providerSchema has all config options, including their descriptions
+	// and types.
+	if providerSchema, err := common.CloudSchemaByType(cloudType); err == nil {
```

Comment:
To avoid excessive indenting, we should make this call outside of the if statement and do the error checks, and then proceed. Same below.